fpga_interchange/xc7_device_timing_patching.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python3
```

Comment:
I wonder whether we want to have this script more generic instead of keeping it specific for xc7. 

Basically the specificity comes in play as the prjxray-db timings dir is organized in a certain way. As a thought, we may keep this patching script generic, but specialize only the functions to retrieve the data, and use the specifiic arch function depending on an argument. Similarly to what happens [here](https://github.com/SymbiFlow/python-fpga-interchange/blob/baa113424c1803704e3123d2c7a298a728e98e3b/fpga_interchange/fasm_generator.py#L36-L47).

Reply:
Fixed in the newest commit